src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -810,8 +810,17 @@ def view_in_browser(resource_group_name, name, slot=None, logs=False):
 
 
 def _open_page_in_browser(url):
-    import webbrowser
-    webbrowser.open(url, new=2)  # 2 means: open in a new tab, if possible
+    import sys
+    if sys.platform.lower() == 'darwin':
```

Comment:
Actually, 3.x is not working well on mac for this scenario, see #3604. I can check the osx version, but that is a bit too much sniffing.